src/sentry/api/endpoints/organization_events_facets_stats_performance.py

```diff
@@ -0,0 +1,123 @@
+from datetime import datetime, timedelta
+from typing import Dict, Optional, Sequence
+
+import sentry_sdk
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases import NoProjects
+from sentry.api.endpoints.organization_events_facets_performance import (
+    OrganizationEventsFacetsPerformanceEndpointBase,
+    query_facet_performance,
+    query_tag_data,
+)
+from sentry.snuba.discover import top_events_timeseries
+
+ALLOWED_AGGREGATE_COLUMNS = {
+    "transaction.duration",
+    "measurements.lcp",
+    "spans.browser",
+    "spans.http",
+    "spans.db",
+    "spans.resource",
+}
+
+TAG_ALIASES = {"release": "sentry:release", "dist": "sentry:dist", "user": "sentry:user"}
+DEFAULT_TAG_KEY_LIMIT = 5
+ONE_DAY = int(timedelta(hours=6).total_seconds())
+
+
+@region_silo_endpoint
+class OrganizationEventsFacetsStatsPerformanceEndpoint(
+    OrganizationEventsFacetsPerformanceEndpointBase
+):
+    def get(self, request: Request, organization) -> Response:
+        try:
+            params, aggregate_column, filter_query = self._setup(request, organization)
+        except NoProjects:
+            return Response([])
+
+        all_tag_keys = None
+        tag_key = None
+
+        all_tag_keys = request.GET.get("allTagKeys")
+        tag_key = request.GET.get("tagKey")
+
+        if tag_key in TAG_ALIASES:
+            tag_key = TAG_ALIASES.get(tag_key)
+
+        with sentry_sdk.start_span(op="discover.endpoint", description="discover_query"):
+            referrer = "api.organization-events-facets-stats-performance.top-tags"
```

Comment:
Should use the referrer module in snuba,
ie. `from sentry.snuba.referrer import Referrer`